<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientProgram</class>
 <widget class="QWidget" name="ClientProgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientProgram</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>主机：</string>
       </property>
       <property name="buddy">
        <cstring>leHost</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leHost">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>用户名：</string>
       </property>
       <property name="buddy">
        <cstring>leUser</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leUser">
       <property name="text">
        <string>gwpl</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbLogin">
       <property name="text">
        <string>登录</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>好友列表</string>
         </property>
         <property name="buddy">
          <cstring>lwContacts</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwContacts">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="enableCtrl" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbInfos">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="teMessages">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leSendBuffer"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
